app/forms/ask_a_question_form.rb

```diff
@@ -0,0 +1,52 @@
+# frozen_string_literal: true
+class AskAQuestionForm
+  include ActiveModel::Model
+  attr_accessor :name, :email, :subject, :message, :location_code, :context, :title
+
+  validates :name, :email, :message, presence: true
+  validates :email, email: true
+
+  def email_subject
+    "[PULFA] #{subject_string}"
+  end
+
+  def subject_string
+    return title if subject == "collection"
+    subject
+  end
+
+  def subject_options
+    [
+      ["This Collection", "collection"],
+      ["Reproductions & Photocopies", "reproduction"],
+      ["Rights & Permissions", "permission"],
+      ["Access", "access"],
+      ["Other", "how much"]
```

Comment:
I am sort of curious what "how much" means

Reply:
Yeah, it seemed weird to me too, but I'm just matching the old values.